doto-frontend/src/components/pages/Settings/SettingsPage.js

```diff
@@ -136,12 +123,40 @@ const SubmitButton = () => {
 };
 
 const SettingsPage = () => {
+    const [theme, setTheme] = useState("blue");
```

Comment:
Will this be used globally - can this be called by other functions? Or does this need right up at the top?

Reply:
No, at the moment themes only work with the settings page